tests/unit/cli/test_utils.py

```diff
@@ -153,6 +153,33 @@ def test_content_gets_decoded(self, content):
+
+    def test_no_payload(self):
+        payload = {}
+        result = utils.format_metadata_query(payload)
+        assert result == ''
+
+    @pytest.mark.parametrize('payload', [{}, None])
+    def test_payload_defined_but_empty(self, payload):
+        result = utils.format_metadata_query(payload)
+        assert result == ''
+
+    def test_metadata_cannot_be_decoded(self):
+        payload = {'metadata': b'\t23hsdf'}
+        result = utils.format_metadata_query(payload)
+        assert result == ''
+
+    @pytest.mark.parametrize('metadata', [_manifest_metadata, _docker_history_metadata])
+    def test_content_gets_decoded(self, metadata):
+        payload = {'metadata': metadata}
+        result = utils.format_metadata_query(payload)
+        assert result != ''
```

Comment:
I get that the result will always be different, but you can still try to assert something that you know is constant. In this case, this test would pass if `result is None` for example, and that would go undetected.

You still have access to the incoming variable so the test could ensure that is included:

```suggestion
        assert metadata in result
```

Reply:
I slightly modified this to make sure the string is formatted, but don't want to mess with b64 decoding during the test, unless you disagree